bigchaindb/crypto/buffer.py

```diff
@@ -0,0 +1,435 @@
+import base64
+import hashlib
+from math import ceil
+
+import binascii
+
+from six import string_types
+
+MSB = 0x80
+REST = 0x7F
+MSBALL = ~REST
+INT = 2 ** 31
+
+# https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER
```

Comment:
> Javascript specs in Python code?
> There is sys.maxint: https://docs.python.org/2/library/sys.html#sys.maxsize

actually, It was intended to keep inline with the js reference library of the interledger protocol
python can handle larger integers than js:
`py:  9223372036854775807`
`js: 9007199254740991`